ppocr/utils/network.py

```diff
@@ -75,6 +75,9 @@ def maybe_download_params(model_path):
     else:
         url = model_path
     tmp_path = os.path.join(MODELS_DIR, url.split('/')[-1])
+    if os.path.exists(tmp_path) and os.path.isfile(tmp_path):
```

Comment:
can be deleted

Reply:
Hi models from **Model_Path** was downloaded to tmp_path but it is repeated to do so each time when we launch a task in the same machine, which takes about 3-5 minutes which is unnecessary.

A better way I think is, by providing MD5 sum in model configuration file, we can check if the files in tmp_path is correct.